### PR TITLE
Extremely Accurate Job Renaming

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
i tweaked a few of the names and descriptions of various jobs to be more accurate to thier true role aboard the ship

## Why It's Bad For The Game
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

## Changelog
:cl:
fix: Fixed names of all jobs
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
i have actually tested this and it works

**Comment by ValoTheValo** [comment]
uhh why it failing now

**Comment by ValoTheValo** [comment]
that fixed it

**Comment by Humonitarian** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up recoil and make all these guns impossible to shoot, thirdly, one and possibly more gun recoil changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by MLGTASTICa** [comment]
Actually against merging this since it can corrupt save files.

**Comment by ValoTheValo** [comment]
> Actually against merging this since it can corrupt save files.

didn't have that issue on my test, also last April fools didn't corrupt save files

**Review by SirRichardFrancis — APPROVED** [review]
Best PR this year. Also tags are untouched, so shit shouldn't break.

**Comment by MLGTASTICa** [comment]
> > Actually against merging this since it can corrupt save files.
> 
> didn't have that issue on my test, also last April fools didn't corrupt save files

did last april fools modify any role_Text and the such?

**Comment by ValoTheValo** [comment]
think thats the last update for this stupid meme. fuck it isnt compiling

**Comment by ValoTheValo** [comment]
oh wait it waas caused by quotes in a copypasta

File: code/game/antagonist/station/inquisitor.dm [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /datum/antagonist/inquisitor
 	id = ROLE_INQUISITOR
-	role_text = "NeoTheology Inquisitor"
-	role_text_plural = "NeoTheology Inquisitors"
-	bantype = ROLE_BANTYPE_INQUISITOR
```
Comment (Firefox13) [inline code/game/antagonist/station/inquisitor.dm:5]: изображение

Comment (ValoTheValo) [reply code/game/antagonist/station/inquisitor.dm:5]: ill revert it then

**Comment by Firefox13** [comment]
This shouldn't break anything because it doesn't modify datums code themselves, only their names

File: code/game/antagonist/station/inquisitor.dm (hunk 1 above)
Comment (ValoTheValo) [reply code/game/antagonist/station/inquisitor.dm:5]: fixed

**Comment by ValoTheValo** [comment]
allright merges are starting that's all folks

**Comment by drexample** [comment]
https://youtu.be/ZazNbG_3m9A

**Comment by ValoTheValo** [comment]
> https://youtu.be/ZazNbG_3m9A

bruh

**Pull request closed without merging**